pkgs/by-name/mo/monophony/package.nix

```
@@ -0,0 +1,78 @@
+{ lib
+, fetchFromGitLab
+, python310Packages
+, wrapGAppsHook4
+, gst_all_1
+, gobject-introspection
+, gtk4
+, yt-dlp
+, libadwaita
+, libsoup_3
+, glib-networking
+,
+}:
+let
+  # pydbus from mpris_server is constrained to python310
+  python3Packages = python310Packages;
+in
+python3Packages.buildPythonApplication rec {
+  pname = "monophony";
+  version = "2.3.0";
+  format = "other";
+
+  sourceRoot = "source/source";
+  src = fetchFromGitLab {
+    owner = "zehkira";
+    repo = "monophony";
+    rev = "v${version}";
+    hash = "sha256-n9tioIXMjUs9AmNuVLrOHJqeGhotkFlv+bTeHlsQPaM=";
+  };
+
+  pythonPath = with python3Packages; [
+    mpris_server
+    pygobject3
+    ytmusicapi
+  ];
+
+  nativeBuildInputs =
+    [
+      gobject-introspection
+      wrapGAppsHook4
+    ]
+    ++ (with python3Packages; [ ordered-set nuitka ]);
+
+  buildInputs =
+    [
+      libadwaita
+      # needed for gstreamer https
+      libsoup_3
+      glib-networking
+    ]
+    ++ (with gst_all_1; [
+      gst-plugins-base
+      gst-plugins-good
+      gstreamer
+    ]);
+
+  makeFlags = [ "prefix=$(out)" ];
```

```suggestion
  installFlags = [ "prefix=$(out)" ];
```
Can we use installFlags instead?

---

Sure

---

Done